<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortChange</class>
 <widget class="QWidget" name="PortChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通道变换测试</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7" columnstretch="6,1">
   <property name="leftMargin">
    <number>13</number>
   </property>
   <property name="topMargin">
    <number>13</number>
   </property>
   <property name="rightMargin">
    <number>13</number>
   </property>
   <property name="bottomMargin">
    <number>13</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="13,5">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="title">
        <string>测试数据</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_test"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="title">
        <string>差值比较</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_compare"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="7,3,2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">font:  12pt &quot;华文楷体&quot;;</string>
       </property>
       <property name="title">
        <string>设置矢网</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2,1">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_start"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_end_freq"/>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="comboBox_end"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>结束频率：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_start_freq"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>开始频率：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>频点数：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_if"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>功率：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>中频：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_power"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_iter"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>dBm</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QComboBox" name="comboBox_if"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>仪器类型：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_type">
          <item>
           <property name="text">
            <string>Agilent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R&amp;S</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>工作状态:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_status">
          <item>
           <property name="text">
            <string>接收</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>发射</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="title">
        <string>测试</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>电子开关：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_elec">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_phase"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>状态：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_clear">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>测试</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;华文楷体&quot;;</string>
       </property>
       <property name="title">
        <string>比较控制</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>A状态：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_A_phase"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>B状态：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_B_phase"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_compare">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>比较</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
